src/olympia/blocklist/templates/blocklist/block_change_form.html

```diff
@@ -4,13 +4,5 @@
 {% block submit_buttons_bottom %}
   {# This is so we can submit directly to the blocksubmission add view - it expect an input_guids field #}
   <input type="hidden" name="input_guids" value="{{ original.guid }}">
-  <div class="submit-row">
```

Comment:
I don't understand this change. I can see that code that is similar to (but not exactly like) this exists in `src/olympia/blocklist/templates/blocklist/block_submission_change_form.html`, but why is this being removed from here. Also, what does `{{ block.super }}` do? Where is `"admin/change_form.html"` that this extends?

Reply:
`{{ block.super }}` means use the parent definition of `{% block submit_buttons_bottom %}` (like `super()` inside a python object function).  `admin/change_form.html` is the original django admin template https://github.com/django/django/blob/2.2.10/django/contrib/admin/templates/admin/change_form.html

I removed my custom template fragment because I realised it wasn't needed any longer.